### DISCUSSION TITLE
ıbaxc referans coin vermedi

### DISCUSSION THREAD

**Discussion by Kriptoportakal** [IBAX-io/go-ibax#655] [Soru Cevap - Türkçe]
refferal coin vermedi neden acaba

**Comment by Rdagci1997** [comment]
Güzel proje sabreden kazanır.

**Comment by Yusufknk** [comment]
Davet ödüllerinin alınması için her iki tarafın görevleri eksiksiz yapması gerekir. Aynı zamanda airdrop bitimine kadar sosyal medya kanallarının takibini bırakmamaları gerekmektedir.
